site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-list.vm

```diff
@@ -40,7 +40,7 @@
 		<div class="sakai-table-toolBar">
 			<div class="sakai-table-filterContainer">
 
-				<form name="viewForm" method="post" action="#toolForm("SiteAction")">
+				<form name="viewForm" class="worksiteSetupViewAndTermFilterForm" method="post" action="#toolForm("SiteAction")">
```

Comment:
There's another component like that on the bottom of the page, it should be updated too.

Reply:
Good catch, @bgarciaentornos! I'll update my PR.